app/src/main/java/com/example/fideicomisoapproverring/WalletSelectionBottomSheet.kt

```
@@ -0,0 +1,61 @@
+            WalletOption("Hana Wallet", false)
+        )
+
+        walletList.adapter = WalletAdapter(wallets) { wallet ->
+            if (wallet.isAvailable) {
+                onWalletSelected(wallet.name)
+                if (wallet.name == "LOBSTR") {
+                    try {
+                        val lobstrUrl = "lobstr://" // Explicit intent for LOBSTR
+                        val intent = Intent(Intent.ACTION_VIEW, Uri.parse(lobstrUrl))
+                        startActivity(intent)
+                    } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)